src/main/scala/devices/spi/TLSPI.scala

```diff
@@ -74,36 +74,36 @@ class SPITopModule(c: SPIParamsBase, outer: TLSPIBase)
   io_int(0) := (ip.txwm && ie.txwm) || (ip.rxwm && ie.rxwm)
 
   protected val regmapBase = Seq(
-    SPICRs.sckdiv -> Seq(RegField(c.divisorBits, ctrl.sck.div)),
-    SPICRs.sckmode -> Seq(
-      RegField(1, ctrl.sck.pha),
-      RegField(1, ctrl.sck.pol)),
-    SPICRs.csid -> Seq(RegField(c.csIdBits, ctrl.cs.id)),
-    SPICRs.csdef -> ctrl.cs.dflt.map(x => RegField(1, x)),
-    SPICRs.csmode -> Seq(RegField(SPICSMode.width, ctrl.cs.mode)),
-    SPICRs.dcssck -> Seq(RegField(c.delayBits, ctrl.dla.cssck)),
-    SPICRs.dsckcs -> Seq(RegField(c.delayBits, ctrl.dla.sckcs)),
-    SPICRs.dintercs -> Seq(RegField(c.delayBits, ctrl.dla.intercs)),
-    SPICRs.dinterxfr -> Seq(RegField(c.delayBits, ctrl.dla.interxfr)),
-
-    SPICRs.fmt -> Seq(
-      RegField(SPIProtocol.width, ctrl.fmt.proto),
-      RegField(SPIEndian.width, ctrl.fmt.endian),
-      RegField(SPIDirection.width, ctrl.fmt.iodir)),
-    SPICRs.len -> Seq(RegField(c.lengthBits, ctrl.fmt.len)),
+    SPICRs.sckdiv -> Seq(RegField(c.divisorBits, ctrl.sck.div, RegFieldDesc("sckdiv","Serial clock divisor", reset=Some(3)))),
```

Comment:
I ddin't check all these reset values. This would be a nice place to use `DescribedReg` to avoid having to make sure they are all correct...

Reply:
also same comment about using whatever names/descriptions are currently in the docs. I didn't check them all